source/read-from-mongodb.txt

```diff
@@ -42,6 +42,18 @@ Overview
 
          .. include:: /scala/filters.txt
 
+.. important:: Change Stream Schema Inference
+
+   When the {+driver-short+} infers the schema of a data frame 
+   read from a change stream, by default,
+   it will use the schema of the underlying collection rather than the schema
+   of the change stream. To instruct the connector to use the schema of the 
+   change stream, set the ``change.stream.publish.full.document.only`` option 
+   to ``true``.
```

Comment:
```suggestion
   of the change stream. The connector infers the schema from the 
   change stream if you set the ``change.stream.publish.full.document.only`` option 
   to ``true``.
```

Reply:
reworded. just for my info, what don't you like about the first version?

Reply:
The phrasing of "instruct the connector" seemed needlessly complicated. Better to just have it be a declarative sentence, IMO. but it just a suggestion :)